common/src/main/java/org/opensearch/ml/common/connector/ConnectorAction.java

```diff
@@ -63,6 +63,9 @@ public ConnectorAction(
         if (method == null) {
             throw new IllegalArgumentException("method can't null");
         }
+        if (requestBody == null) {
+            throw new IllegalArgumentException("request body can't null");
```

Comment:
`request body can't be null`

Reply:
Yes, I just followed existing error message text, will update it everywhere after we decide on above point.

Reply:
@akolarkunnu @dhrubo-os Actually request body can be null sometimes. We don't pass in body for few requests like GET/CANCEL: https://opensearch.org/docs/latest/ml-commons-plugin/api/model-apis/batch-predict/#prerequisites

Lets remove this null check here.

Reply:
Removed the null check for "requestBody"